requirements.txt

```diff
@@ -1,5 +1,5 @@
 aiohttp
-gunicorn
 pymongo
+python-dateutil
```

Comment:
where is this used, should this be also in `setup.py` ?

why no more `gunicorn` ?

Reply:
Dateutil was added for [adding publish dates for studies](https://github.com/CSCfi/metadata-submitter/blob/origin/add_more_endpoints_and_queryable_submissions/metadata_backend/api/parser.py#L97) (I think ENA uses +2 months from submission date by default). I also think some date-related parsing is needed later to handle those HOLD/SUPPRESS etc actions, so this library is probably useful.

I recall we thought that gunicorn is not needed. But anyways, it should be removed from [deploy/app.sh](https://github.com/CSCfi/metadata-submitter/blob/origin/add_more_endpoints_and_queryable_submissions/deploy/app.sh#L7) as well, if it it's not used. What do you think?

Reply:
And setup.py gets parses requirements directly from requirements.txt: https://github.com/CSCfi/metadata-submitter/blob/master/setup.py#L5

Reply:
ok to both :)
In `deploy/app.sh` we need to see. I would still prefer to have it there for the moment, but don't remember exactly if the s2i container installs it automatically or not.